Models/Animation Validation/MachinesValidation.hmf

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <hopsanmodelfile hmfversion="0.4" hopsanguiversion="0.6.x_r7435" hopsancoreversion="0.6.x_r7431">
   <requirements>
-    <componentlibrary>DigitalHydraulicsLib</componentlibrary>
     <componentlibrary>HopsanDefaultComponentLibrary</componentlibrary>
```

Comment:
Why not use the library ID here, now that the default lib has one?

Reply:
I made this change before adding ID. Then I considered updating the models again. But decided not to (to much  work). This informatin is still valid for models saved with an older version of hopsan. The load code will show a debug message if the library name is not pressent. (Like before). Only difference is that now it wont warn for non needed libraries. 
The next time the models are updated (resaved) the new ID info will be added.